package.json

```diff
@@ -10,8 +10,9 @@
   "scripts": {
     "prebuild": "npm run clean",
     "build": "babel src --out-dir lib --copy-files",
+    "precommit": "pretty-quick --staged",
     "clean": "rimraf lib/",
-    "lint": "eslint src/ test/",
+    "lint": "xo 'src/**/*.{js,ts}' 'test/**/*.{js,ts}'",
     "prepublish": "npm test && npm run build",
```

Comment:
`prepublish` can be changed to `prepublishOnly`: https://docs.npmjs.com/misc/scripts#prepublish-and-prepare. I also think that we don't need to test here when publishing from `master` as it should always be stable.